libraries/AdaptiveExpressions/AdaptiveExpressions.csproj

```diff
@@ -20,7 +20,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>Microsoft.Bot.Expressions</PackageId>
+    <PackageId>AdaptiveExpressions</PackageId>
```

Comment:
This needs to be Adaptive.Expressions with a period.  Adaptive already has a number of a packages which use the dot nomenclature
Adaptive.WPF
Adaptive.Xaml
etc.

Reply:
Hmmm..Actually, they are AdaptiveCards.WPF, AdaptiveCards.Xaml etc. 
Do we want Adaptive or AdaptiveCards?

---
In reply to: [377424557](https://github.com/microsoft/botbuilder-dotnet/pull/3375#discussion_r377424557) [](ancestors = 377424557)

Reply:
@vishwacsena maybe we should confirm this?

Reply:
yes, let @vishwacsena  to confirm, i thought we should call this AdapativeExpression